### DISCUSSION TITLE
📢 Let's decide on a new repo name

### DISCUSSION THREAD

**Discussion by madskjeldgaard** [madskjeldgaard/portedplugins#20] [Ideas]
Hello everyone

I'd like to change the name of this repository and I am open to suggestions. Please post them here if you any ideas!

This repo started out as a few personal experiments but has quickly grown to something bigger and I'd like for it to be reflected in the name of it that it is a community project and not a personal project.

The general theme for these plugins is that they are ports (for the moment at least) of other people's work - maybe that's something that should be reflected in the name?

## Candidate names

- Ported Plugins
- SuperCollider Ports
- Visitors
- Sprawl
- The trunk

**Comment by danielmkarlsson** [comment]
Seashells

From the Cage story about walking along a beach in awe of the breathtaking beauty present in each new seashell you find.

**Comment by kflak** [comment]
Armada, as in the pirate city in China Miéville's The Scar: thousands upon thousands of ships latched together to form an artificial island, a floating city state...
